app/helpers/ugc_form_helper.rb

```diff
@@ -31,4 +31,27 @@ def contribution_state_change_input(builder, events, selected)
   def aasm_events_for_select(events)
     events.map { |event| [t(event.name, scope: 'contribute.contributions.events'), event.name] }
   end
+
+  ##
+  # To look up the mongo documents for:
+  # https://creativecommons.org/publicdomain/mark/1.0/
+  # https://creativecommons.org/licenses/by-sa/4.0/
+  # http://rightsstatements.org/page/CNE/1.0/
+  def edm_rights_options
+    [
+      [edm_rights_label_html('public_domain'), rights_id_from_url('http://creativecommons.org/publicdomain/mark/1.0/')],
```

Comment:
I am reluctant to have to maintain a hard-coded mapping of I18n keys to license URIs. Let's derive the former from the latter instead:
```ruby
def cc_license_i18n_key(rdf_about)
  uri = URI.parse(rdf_about)
  uri.host.tr('.', '_') + uri.path.tr('/.', '._').sub(/\.\z/, '')
end

CC::License.each { |l| puts cc_license_i18n_key(l.rdf_about) }
```
```
creativecommons_org.publicdomain.mark.1_0
rightsstatements_org.vocab.NoC-NC.1_0
rightsstatements_org.vocab.NoC-OKLR.1_0
creativecommons_org.publicdomain.zero.1_0
creativecommons_org.licenses.by.4_0
creativecommons_org.licenses.by-sa.4_0
creativecommons_org.licenses.by-nd.4_0
creativecommons_org.licenses.by-nc.4_0
creativecommons_org.licenses.by-nc-sa.4_0
creativecommons_org.licenses.by-nc-nd.4_0
rightsstatements_org.vocab.InC.1_0
rightsstatements_org.vocab.InC-EDU.1_0
rightsstatements_org.vocab.InC-OW-EU.1_0
rightsstatements_org.vocab.CNE.1_0
```